curated_transformers/repository/fsspec.py

```diff
@@ -97,15 +107,47 @@ def __init__(
 
     def file(self, path: str) -> RepositoryFile:
         full_path = f"{self.repo_path}/{path}"
-        if not self.fs.exists(full_path, **self.fsspec_args.kwargs):
-            raise FileNotFoundError(f"Cannot find file in repository: {path}")
         return FsspecFile(self.fs, full_path, self.fsspec_args)
 
     def pretty_path(self, path: Optional[str] = None) -> str:
         if not path:
             return self._protocol
         return f"{self._protocol}/{path}"
 
+    def transaction(self) -> TransactionContext:
+        return FsspecTransactionContext(self)
+
     @property
     def _protocol(self) -> str:
         return self.fs.unstrip_protocol(self.repo_path)
+
+
+class FsspecTransactionContext(TransactionContext):
+    """
+    `fsspec`_ transaction context manager.
+    TODO: Implement/currently a noop
```

Comment:
Should the TODO be in here? How is this a noop?

Reply:
Yeah, it's intended. The transaction semantics of `fsspec` are different from how we handle it for HF repos, so we need to think about it a bit more. The stub class is just there to aid the test suite/CI.